n64.ld

```diff
@@ -28,8 +28,15 @@ MEMORY
 }
 
 SECTIONS {
+   /* Start address of code is 1K up in uncached, unmapped RAM.  We have
```

Comment:
Good documentation!

Reply:
This was already present in one of linker scripts indeed.